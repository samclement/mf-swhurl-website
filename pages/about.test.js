/* eslint-env jest */

import React from 'react'
import { render, shallow, mount } from 'enzyme'

import About from './about.js'

describe('With Snapshot Testing', () => {
  it('About page shows "About" H1', () => {
      const wrapper = render(<About/>)
      expect(wrapper).toMatchSnapshot()
    })
})

