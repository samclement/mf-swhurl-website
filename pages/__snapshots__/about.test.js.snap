// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing About page gets correct data 1`] = `
Object {
  "completed": false,
  "id": 1,
  "title": "delectus aut autem",
  "userId": 1,
}
`;

exports[`With Snapshot Testing About page shows "About" H1 1`] = `
.c1 {
  color: deeppink;
}

.c0 {
  color: 'deeppink';
}

<div>
  <nav>
    <a
      class="c0"
      href="/"
    >
      Home
    </a>
     | 
    <a
      class="c0"
      href="/about"
    >
      About
    </a>
     | 
    <a
      class="c0"
      href="/contact"
    >
      Contact
    </a>
     | 
    <a
      class="c0"
      href="/login"
    >
      Login
    </a>
  </nav>
  <h1
    class="c1"
  >
    About
  </h1>
  <p />
</div>
`;
